html/Linkbrary(signup).html

```diff
@@ -1,67 +1,72 @@
 <!DOCTYPE html>
-<html style="font-size: 10px;">
-  <head>
+<html lang="ko" style="font-size: 10px;">
+<head>
     <meta charset="utf-8">
     <title>Linkbrary</title>
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link
-      href="https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@400;700&family=Titillium+Web:wght@300;600&display=swap"
-      rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@400;700&family=Titillium+Web:wght@300;600&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="/css/linkbrary_signup.css">
-  </head>
-  <body>
+</head>
+<body>
     <div class="main">
-    <logo class="logo">
-      <a href="/html/index.html" target="_self">
-      <img class="logoimg" src="/image/logo.png">
-      </a>
-      <div class="logoTitle">
-      <h1 class="logotitle">이미 회원이신가요?</h1>
-      <a href="/html/Linkbrary(signin).html" target="_self">
-      <h1 class="logotitle signup">로그인 하기</h1>
-      </a>
-      </div>
-    </logo>
-    <form>
-      <div class="email">
-        <label for="email">이메일</label>
-        <input id="email" name="email" type="email">
-      </div>
-      <div class="password">
-        <label for="password">비밀번호</label>
-        <input id="password" name="password" type="password">
-        <img class="eye" src ="/image/eye-on.png">
-      </div>
-      <div class="password">
-        <label for="password">비밀번호 확인</label>
-        <input id="password" name="password-repeat" type="password">
-        <img class="eye" src ="/image/eye-on.png">
-      </div>
-    </form>
-    <div>
-      <div class="boxmain">
-        <h1>회원가입</h1>  
-    </div>
-    </div>
+        <logo class="logo">
+            <a href="/index.html" target="_self">
+                <img class="logoimg" src="/image/logo.png">
```

Comment:
r: img 태그에는 [접근성](https://tech.kakaopay.com/post/accessibility-stories-for-everyone/) 및 이미지 요소 로드 실패 처리를 위해 alt 속성을 지정하는 것이 좋습니다.

```suggestion
                <img class="logoimg" src="/image/logo.png" alt="linkbrary 로고">
```